README.md

```diff
@@ -79,13 +99,19 @@ services:
 
 ### With extension-installer
 
-Unfortunately, by using `phpstan/extension-installer` you can't enable it one by one, ***but you can [ignore specific errors](https://phpstan.org/user-guide/ignoring-errors) instead***.
+When using `phpstan/extension-installer` you can disable automatic rules loading by:
+
+```neon
+parameters:
+	strictRules:
+		enabled: false
```

Comment:
I'd really like my original suggestion of `turnOffAllRules.neon` instead, this one reads really weird...

Reply:
What about using better name instead of enabled? From my user PoW it is more convenient to set parameter than including file from depths of vendor.

But if you really preffer only file I will change it.

Reply:
What about `allRules`, `enableAllRules`, `useAllRules`, `turnOnAllRules`?

Reply:
Yeah, `allRules` parameter would be fine 👍

Reply:
Updated